document/CoreFormat.mdk

```diff
@@ -508,6 +552,43 @@ whose value is a string specifying that path.
 If the location of the file within its containing property cannot be expressed by means of a path,
 then the `pathFromParent` property shall be absent.
 
+If the `file` object occurs as the only member of an array within the `run.files` object (&sect;[#run-files]),
+then the `pathFromParent` property may be absent.
+If it is present, it must be equal to the corresponding property name within the `run.files` object,
+
+~ IsoNote
+In this case, it might be interpreted as a path from a "null" parent.
```

Comment:
And then you can drop this case, since an absolute uri stands on its own without explanation...